searcch_backend/api/common/scheduled_tasks.py

```diff
@@ -0,0 +1,44 @@
+import atexit
+from searcch_backend.api.app import db
+from searcch_backend.models.model import Sessions, StatsRecentViews, StatsArtifactViews
+from apscheduler.schedulers.background import BackgroundScheduler
```

Comment:
Also, how does this mesh with gunicorn multi-process, or eventlet green threads, etc -- what is the execution model?  Is there a background task per gunicorn process, for instance?

Reply:
I hadn't really thought about that David. Thank you for bringing it up. Apparently gunicorn replicates the APS background process for every worker instance; but upon looking up online, I found that there is a way to ensure only one background process runs (just on the master gunicorn process). I can work on implementing the same too.